v1/serviceEntity/stores/github-store-config.yaml

```diff
@@ -0,0 +1,16 @@
+title: GithubStoreConfig
+allOf:
+- $ref: basic-manifest-store-config.yaml
+- type: object
+  required:
+  - connector
+  - location
+  properties:
+    location:
```

Comment:
please add description for location field

Reply:
done.